package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "nextjs-template",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prepare": "husky install"
+  },
+  "dependencies": {
+    "@chakra-ui/react": "^1.8.1",
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@hookform/resolvers": "^2.8.8",
+    "framer-motion": "^5.6.0",
+    "lodash": "^4.17.21",
+    "next": "12.0.10",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-hook-form": "^7.25.3",
```

Comment:
Although I know you need the form dependency to build the starter form, is it right to have the starter depend on react-hook-form and yup? No shade at these libs but that seems like a decision I would pass to the project

Reply:
I agree with Jordaniza on excluding these two as they are more project specific.

Reply:
I also speak from experience that keeping lodash up to date can be a real pain with all the security vulnerabilities it gets flagged for.